src/Entity/User.php

```diff
@@ -35,6 +35,10 @@ class User implements UserInterface
     /**
      * @var string The hashed password
      * @ORM\Column(type="string")
+     * @Assert\Length(
+     * min = 6,
+     * minMessage = "Votre mot de passe doit faire au moins 6 caractères",
+     * )
```

Comment:
Caution : do not mix the hashed password and the clear password.
The hashed password will never be less than 6 (and probably a lot more.... like 32) but it does not depend on the length of the clear password.